build.gradle

```diff
@@ -414,6 +414,7 @@ project(':iceberg-data') {
     useJUnitPlatform()
     // Only for TestSplitScan as of Gradle 5.0+
     maxHeapSize '1500m'
+    jvmArgs += '-XX:-OmitStackTraceInFastThrow'
```

Comment:
I'm not familiar with Gradle syntax. Let me know if there is a better way. 
Also, I wonder if we should add the same argument to the `integrationTest` task.

Reply:
is there a particular reason why this is only being applied to `iceberg-data` and not around L230? Also I think this is good to apply to `integrationTest` as well, but I would add a small comment describing why it's being added to both places

Reply:
It was my oversight. I think we can use `tasks.withType(Test)` to avoid duplication.